test/fixtures/test-runner/output/test-timeout-flag.snapshot

```diff
@@ -0,0 +1,55 @@
+TAP version 13
+# Subtest: --test-timeout is set to 20ms
+    # Subtest: should timeout after 20ms
+    not ok 1 - should timeout after 20ms
+      ---
+      duration_ms: *
+      type: 'test'
+      location: '/test/fixtures/test-runner/output/test-timeout-flag.js:(LINE):3'
+      failureType: 'testTimeoutFailure'
+      error: 'test timed out after 20ms'
+      code: 'ERR_TEST_FAILURE'
+      stack: |-
+        async Promise.all (index 0)
+      ...
+    # Subtest: should timeout after 5ms
+    not ok 2 - should timeout after 5ms
+      ---
+      duration_ms: *
+      type: 'test'
+      location: '/test/fixtures/test-runner/output/test-timeout-flag.js:(LINE):3'
+      failureType: 'testTimeoutFailure'
+      error: 'test timed out after 5ms'
+      code: 'ERR_TEST_FAILURE'
+      ...
+    # Subtest: should not timeout
+    ok 3 - should not timeout
+      ---
+      duration_ms: *
+      type: 'test'
+      ...
+    # Subtest: should pass
+    ok 4 - should pass
+      ---
+      duration_ms: *
+      type: 'test'
+      ...
+    1..4
+not ok 1 - --test-timeout is set to 20ms
+  ---
+  duration_ms: *
+  type: 'suite'
+  location: '/test/fixtures/test-runner/output/test-timeout-flag.js:(LINE):1'
+  failureType: 'subtestsFailed'
+  error: '2 subtests failed'
+  code: 'ERR_TEST_FAILURE'
+  ...
+1..1
+# tests 4
+# suites 1
+# pass 2
+# fail 0
+# cancelled 2
+# skipped 0
+# todo 0
```

Comment:
While I suppose this is already the "as is", I have mixed feelings about the `cancelled 2` / `fail 0`
WDYT?

Reply:
Likewise, they are supposed to be failed but on the flip side they are indeed just being cancelled due to timeout (which means the test either didn't get a chance to run a) the parent get timed out or b) the test didn't finish).

Luckily the tests output clearly outline that `test timed out for *`. Maybe it is something we just have to live with 😄

Reply:
Ye I guess so 😊